src/lib/rest/ConnectionInfo.h

```diff
@@ -160,6 +164,7 @@ class ConnectionInfo
   std::string                ip;
   std::string                apiVersion;
   RequestType                requestType;
+  StringFilter*              stringFilterP;
```

Comment:
After skype dicussion it seems this field is not needed. Try to remove it and let's see were it crashes :)